core/src/com/theironyard/MyGdxGame.java

```diff
@@ -0,0 +1,123 @@
+package com.theironyard;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.scenes.scene2d.utils.DragAndDrop;
+
+public class MyGdxGame extends ApplicationAdapter {
+    private TextureRegion down, up, right, left;
+	private SpriteBatch batch;
+	private Texture img;
+    String look = "";
+    private float time;
+
+
+
+	private float x, y, xv, yv = 0;
+	public static final float MAX_VELOCITY = 100;
+
+	@Override
+	public void create () {
+		Texture tiles = new Texture("tiles.png");
+		TextureRegion[][] grid = TextureRegion.split(tiles, 16, 16);
+        down = grid[6][0];
+		up = grid[6][1];
+		right = grid[6][3];
+		left = new TextureRegion(right);
+		left.flip(true, false);
+		batch = new SpriteBatch();
+	}
+
+	@Override
+	public void render () {
+        move();
+        TextureRegion img = down;
+        time += Gdx.graphics.getDeltaTime();
+
+        Gdx.gl.glClearColor(0, .6f, 0, 1);
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+
+		batch.begin();
+        batch.draw(img, x, y);
+
+        if (look == "r"){
```

Comment:
Comparing strings is always a tricky thing for systems to do.  I would recommend not using random strings as values to switch on like this.  One way is to use an `enum` as this is a case of what they were used for.  Alternatively, you can create 4 constant variables that are each integers.

```java
public static final int UP = 1;
public static final int DOWN = 2;
public static final int LEFT = 3;
public static final int RIGHT = 4;
```
This not only is more efficient, it makes the code more readable and takes up less space.  It will look like this
`else if (faceDirection == LEFT)`.  Much more readable and prevents spelling errors as you use the variables.